src/containers/App/App.scss

```diff
@@ -23,30 +23,38 @@ body,
 }
 
 :root {
-    --yc-color-infographics-yellow-light: rgba(255, 199, 0, 0.15);
-    --yc-color-infographics-yellow-medium: rgba(255, 219, 77, 0.4);
-    --yc-color-base-warning-orange: #ff922e;
+    --g-color-base-yellow-light: rgba(255, 199, 0, 0.15);
+    --g-color-base-yellow-medium: rgba(255, 219, 77, 0.4);
 
     --data-table-row-height: 40px;
 }
 
-.yc-root {
-    --ydb-data-table-color-hover: var(--yc-color-base-float-hover);
+.g-root {
+    --ydb-data-table-color-hover: var(--g-color-base-float-hover);
+
+    // Colors for tablets, status icons and progress bars
+    --ydb-color-status-grey: var(--g-color-base-neutral-heavy);
+    --ydb-color-status-green: var(--g-color-base-positive-heavy);
+    --ydb-color-status-yellow: var(--g-color-base-warning-heavy);
+    --ydb-color-status-orange: #ff922e;
+    --ydb-color-status-red: var(--g-color-base-danger-heavy);
+    --ydb-color-status-blue: var(--g-color-base-info-heavy);
+    --ydb-color-status-black: var(--g-color-base-misc-heavy);
```

Comment:
Assigned most common colors to intermediary variables to make it possible to update them more easily